feel/feelvf/ginac.cpp

```diff
@@ -153,11 +153,16 @@ parse( std::string const& str, std::string const& seps, std::vector<symbol> cons
     CHECK( fsize  > 0 ) << "bad expression format";
     std::string strexpr( fields[0] );
     std::vector<std::string> strsyms;
+#if 0
```

Comment:
remove the code here. do you think it will still serve some purpose in the future?